R/helpers.R

```diff
@@ -1003,7 +1003,7 @@ cell_style_structure <- function(name, obj, subclass = name) {
 #' @export
 pct <- function(x) {
 
-  if (!inherits(x, "numeric")) {
+  if (!inherits(x, "numeric") && !inherits(x, "integer")) {
```

Comment:
Maybe 

```r
if (mode(x) != "numeric") {
```

Reply:
Yeah, this is better. Made that change.

Reply:
Minor, perhaps, but isn't `!is.numeric(x)` faster (6-10x for this scale, likely not a real factor), just as declarative, and sufficiently-encompassing that it would work here, too? Asked another way, what is better about `inherits` or `mode` in this case?

Reply:
I think it might be faster and more correct. For example:

```r
# We wouldn't want a `Date` as input
Sys.Date() %>% mode()
#>  "numeric"

Sys.Date() %>%  typeof()
#> "double"

# This is better
Sys.Date() %>% is.numeric()
#> FALSE
```

@schloerke does this warrant a quick look? It does seem like `is.numeric()` allows `double` and `integer` storage types while also excluding classes like `Date` that also store as `double`.

Reply:
I had looked at everything (I thought) *except* `POSIXt` and `Date`, glad it still works. :-)

Reply:
I like `is.numeric()`. I think that's what I was looking for earlier.

Thanks @rich-iannone and @r2evans

Reply:
Great! I’ll open another PR with this small fix a bit later.